src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -906,6 +916,43 @@ private static bool AddFlags(TypeSymbol type, ArrayBuilder<bool> transformFlagsB
 
                 // Continue walking types
                 return false;
+
+                static void handleFunctionPointerType(FunctionPointerTypeSymbol funcPtr, ArrayBuilder<bool> transformFlagsBuilder, bool addCustomModifierFlags)
+                {
+                    Func<TypeSymbol, (ArrayBuilder<bool>, bool), bool, bool> visitor =
+                        (TypeSymbol type, (ArrayBuilder<bool> builder, bool addCustomModiferFlags) param, bool _) => AddFlags(type, param.builder, isNestedNamedType: false, param.addCustomModiferFlags);
```

Comment:
>false [](start = 166, length = 5)

Always passing false doesn't feel right.

Reply:
It doesn't look like this fix is covered by a test.

---
In reply to: [424088267](https://github.com/dotnet/roslyn/pull/44078#discussion_r424088267) [](ancestors = 424088267)

Reply:
@AlekseyTs test added.

---
In reply to: [424722117](https://github.com/dotnet/roslyn/pull/44078#discussion_r424722117) [](ancestors = 424722117,424088267)